README.md

```diff
@@ -418,34 +505,36 @@ For `my` and `as` capabilities limited to some scheme, the action MUST be one no
 {"with": "my:dns", "can": "*"}
 ```
 
-## Proof Field Redelegation
+## 4.3 Proof Field Redelegation
 
-### `prf` Scheme
+### 4.3.1 `prf` Scheme
 
 The `prf` URI scheme defines addressing for UCANs and their fields.
 
 ``` abnf
 prf = "prf:" selector
-selector = "*" / 1*DIGIT
+selector = "*" / cid
 ```
 
-`prf:*` represents all of the UCANs in the current proofs array. The witnesses for the current UCAN MAY be referenced by their index in the `"prf"` field. If selecting a particular witness (i.e. not the wildcard), then zero-based indexing MUST be used. For example, the first UCAN would be selected by `prf:0`, the second by `prf:1`, etc. Under the indexing scheme, selections MUST be performed on the current UCAN only and cannot recurse on nested witnesses.
-
-A further selection of capabilities inside specific witnesses MUST NOT be a valid parsing of this URI. For instance, `prf:0:mailto` MUST NOT be a valid `prf` URI.
+`prf:*` represents all of the UCANs in the current proofs array. The proofs for the current UCAN MAY be referenced by their index in the `"prf"` field. If selecting a particular proof (i.e. not the wildcard), then the [content address](#56-content-identifiers) MUST be used. In the case of selecting a particular proof, the validator MUST check that the delegated content address is listed in the proofs.
 
 ### 4.3.2 `prf` Actions
 
-The `prf` scheme MUST accept the following action: `ucan/DELEGATE`. This action redelegates all of the capabilities in the selected witness(es).
+The `prf` scheme MUST accept the following action: `ucan/DELEGATE`. This action redelegates all of the capabilities in the selected proof(s).
 
-`ucan/delegate` is distinct from the superuser ability and acts as a re-export of the ability. If an attenuated resource or capability is desired, it MUST be explicitly listed without the `prf` URI scheme.
+`ucan/delegate` is distinct from the superuser ability and acts as a re-export of the selected abilities. If an attenuated resource or capability is desired, it MUST be explicitly listed without the `prf` URI scheme.
 
 ``` js
-{ 
-  "with": "prf:7", // Contains: { "with": "mailto:boris@example.com", "can": "email/send" }
-  "can": "ucan/DELEGATE"
-} 
-
-{ "with": "prf:*", "can": "ucan/DELEGATE" }
+[
+  { 
+    "with": "prf:bafkreihogico5an3e2xy3fykalfwxxry7itbhfcgq6f47sif6d7w6uk2ze",
+    "can": "ucan/DELEGATE"
```

Comment:
Why not stay consistent and use `"can": "*"` here just like with `my`, If there is a difference between `*` and `ucan/DELEGATE` it's lost on me and would love if either:

1. This was consistent (would prefer `*` but other way round is fine as well).
2. There was some explanation of how "ucan/DELEGATE" differs from "*".

Reply:
@Gozala The rationale is that `can: *` implies superuser rights. If you can only delegate read-only permissions, this could be confusing.

Reply:
Ah right, this was ambiguous in the previous version, so this is us disambiguating. We need to force one or the other (or both!) to have a longer tag, basically.

Reply:
I'm sorry but I still don't understand the difference, my interpretation of `*` is basically match on everything and as such:

1. `{ with: "my:*", can: "*" }` implies delegate all of my capabilities.
2. `{ with: "prf:bafkreihogico5an3e2xy3fykalfwxxry7itbhfcgq6f47sif6d7w6uk2ze", can: "*" }` implies delegate all the capabilities from linked proof.

Likewise it would make sense to me to allow something like

3. `{ with: "prf:bafkreihogico5an3e2xy3fykalfwxxry7itbhfcgq6f47sif6d7w6uk2ze", can: "email/SEND" }` which would imply all the capabilities from linked proof that provide spelled out ability.

Alternatively you could probably just allow leaving out `can` in `my:`, `prf:`, `as:` schemas would would mean all the `can` from those. Either way both `*` and `ucan/DELEGATE` both seem like selectors on ability that select all.

Reply:
Yes, that was the original interpretation of `*`: "whatever is in scope". Now it means "superuser" (i.e. can do anything, whatever the maximum is for that ability).

@matheus23 I could be misremembering, but I believe that you had initially raised this ambiguity. Any thoughts?

Reply:
i also see `*` as `ANY` action for the resource.

Since 
> If an attenuated resource or capability is desired, it MUST be explicitly listed without the ucan URI scheme.

We could just omit the `can`

Reply:
I personally would prefer omition of `can` to mean any action both here, in `my:*` and everywhere else. Only concern I'd have with omition is that occurring unintentionally (program bug or something) for that reason `can: "*"` would work be less error prone and more consistent with use of `*` everywhere else.